### DISCUSSION TITLE
Library compatibility with nRF52x

### DISCUSSION THREAD

**Discussion by derong97** [feilipu/Arduino_FreeRTOS_Library#77] [Q&A]
Your library seems to support AVR boards only. Can it support nRF52x?

**Comment by feilipu** [comment]
Sorry. No it doesn't.
It does what it says on the box.
[Try here](https://visualgdb.com/tutorials/arm/nrf51/freertos/) though. Or [here at `platform.io`](https://docs.platformio.org/en/latest/platforms/nordicnrf52.html?highlight=nrf52).
